Main/ClientApp/src/Actions/Actions.ts

```diff
@@ -9,3 +11,17 @@ export async function loadCollections() {
     payload: { collections: collections }
   });
 }
+
+export async function addCollection(
+  collectionCreationData: CollectionCreationData
+) {
+  CollectionsApi.addCollection(collectionCreationData);
+  Dispatcher.dispatch({
+    actionType: ActionTypes.ADD_COLLECTION,
+    payload: { collectionCreationData: collectionCreationData }
```

Comment:
ADD_COLLECTION doesn't actually update any component's state, so this dispatch call should be ommited. Insstead, we should wait for response from the server, and if it is 20X we shoukd call the loadCollections actions and it will take care of the rest.